rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -764,12 +1072,80 @@ size_t MasterGraph::bounding_box_batch_count(int *buf, pMetaDataBatch meta_data_
     size_t size = 0;
     for(unsigned i = 0; i < _user_batch_size; i++)
     {
-        buf[i] = _is_box_encoder? _num_anchors: meta_data_batch->get_bb_labels_batch()[i].size();
+        buf[i] = _is_box_encoder? _num_anchors: meta_data_batch->get_labels_batch()[i].size();
         size += buf[i];
     }
     return size;
 }
 
+rocalTensorList * MasterGraph::labels_meta_data()
+{
+    if(_ring_buffer.level() == 0)
+        THROW("No meta data has been loaded")
+    auto meta_data_buffers = (unsigned char *)_ring_buffer.get_meta_read_buffers()[0]; // Get labels buffer from ring buffer
+    for(unsigned i = 0; i < _labels_tensor_list.size(); i++)
+    {
+        _labels_tensor_list[i]->set_mem_handle((void *)meta_data_buffers); // TODO - Need to update according to the metadata
+        meta_data_buffers += _labels_tensor_list[i]->info().data_size();
+    }
+
+    return &_labels_tensor_list;
+}
+
+rocalTensorList * MasterGraph::bbox_labels_meta_data()
```

Comment:
The API labels_meta_data and bbox_labels_meta_data are almost similar, We can check and remove one of them if it is redundant

Reply:
Removed bbox_labels_meta_data